src/test/java/it/gov/pagopa/pu/bff/connector/AuthClientImplTest.java

```diff
@@ -25,27 +25,27 @@
 class AuthClientImplTest {
 
   @Mock
-  private RestTemplateBuilder restTemplateBuilder;
+  private RestTemplateBuilder restTemplateBuilderMock;
 
   @Mock
-  private AuthnApi authApi;
+  private AuthnApi authApiMock;
 
   @InjectMocks
-  private AuthClientImpl authClient;
+  private AuthClientImpl authClientMock;
```

Comment:
This is not a mock, this is the tested class!

Reply:
updated